stl/inc/xpolymorphic_allocator.h

```diff
@@ -229,6 +237,49 @@ namespace pmr {
             _Uses_allocator_construct(_Ptr, _Al, *this, _STD forward<_Types>(_Args)...);
         }
 
+#if _HAS_CXX20

```

Comment:
`construct` appears after the declarations of these new functions in [mem.poly.allocator.class.general]/2. Could we please move it after the `#endif` so everything is in synopsis order?